modules/installing-private-image-registry-private-azure.adoc

```diff
@@ -0,0 +1,121 @@
+      - Microsoft.Storage/storageAccounts/listKeys/action
+      - Microsoft.Resources/tags/write
+      - Microsoft.Network/privateEndpoints/write
+      - Microsoft.Network/privateEndpoints/read
+      - Microsoft.Network/privateEndpoints/privateDnsZoneGroups/write
+      - Microsoft.Network/privateEndpoints/privateDnsZoneGroups/read
+      - Microsoft.Network/privateDnsZones/read
+      - Microsoft.Network/privateDnsZones/write
+      - Microsoft.Network/privateDnsZones/join/action
+      - Microsoft.Network/privateDnsZones/A/write
+      - Microsoft.Network/privateDnsZones/virtualNetworkLinks/write
+      - Microsoft.Network/privateDnsZones/virtualNetworkLinks/read
+      - Microsoft.Network/virtualNetworks/subnets/join/action
+      - Microsoft.Network/virtualNetworks/join/action
+      - Microsoft.Network/networkInterfaces/read
+      - Microsoft.Storage/storageAccounts/PrivateEndpointConnectionsApproval/action
+----
+
+. If you have not previously created installation manifest files, do so by running the following command:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir <installation_directory>
+----
++
+This command displays the following messages:
++
+.Example output
+[source,terminal]
+----
+INFO Consuming Install Config from target directory
+INFO Manifests created in: <installation_directory>/manifests and <installation_directory>/openshift
+----
+
+. Create an image registry configuration object and pass in the `networkResourceGroupName`, `subnetName`, and `vnetName` provided by Microsoft Azure. For example:
++
+[source,terminal]
+----
+$ touch imageregistry-config.yaml
+----
++
+[source,yaml]
+----
+apiVersion: imageregistry.operator.openshift.io/v1
+kind: Config
+metadata:
+  name: cluster
+spec:
+  managementState: "Managed"
+  replicas: 2
+  rolloutStrategy: RollingUpdate
+  storage:
+    azure:
+      networkAccess:
+        internal:
+          networkResourceGroupName: <vnet_resource_group> <1>
+          subnetName: <subnet_name> <2>
+          vnetName: <vnet_name> <3>
+        type: Internal 
+----
+<1> Replace `<vnet_resource_group>` with the resource group name that contains the existing virtual network (VNet).
+<2> Replace `<subnet_name>` with the name of the existing subnet within the specified resource group.
+<3> Replace `<vnet_name>` with the name of the existing virtual network (VNet) in the specified resource group.
+
+. Move the `imageregistry-config.yaml` file to the `<installation_directory/manifests>` folder by running the following command:
++
+[source,terminal]
+----
+$ mv imageregistry-config.yaml <installation_directory/manifests/>
+----
+
+.Next steps
+
+* After you have moved the `imageregistry-config.yaml` file to the `<installation_directory/manifests>` folder and set the required permissions, proceed to "Deploying the cluster". You can also ensure that a private endpoint resource has been generated by following the _Verification_ steps found in the same section.
```

Comment:
1. I think user can still continue the following procedures ( providing short-term or long-term credentials when setting credentialsMode: Manual) instead of proceeding to "Deploying the cluster" directly, right?
2.  what does the _Verification_ steps here refer to?

Reply:
I found _Verificaiton_ step, but seems that it is present in all azure installation doc, this step should only be in private cluster, right?

Reply:
Yes. Fixed. Thanks @jinyunma